pkg/controller/sinks/sinks_test.go

```diff
@@ -32,6 +32,7 @@ import (
 
 var (
 	addressableDNS = "addressable.sink.svc.cluster.local"
+	addressableURL = fmt.Sprintf("http://%s/", addressableDNS)
```

Comment:
In #477 I was doing a _straight_ port of what @mattmoor did,  so, only updating the 

```
			want:      fmt.Sprintf("http://%s", addressableDNS),
```

Note, that the format here is different `%s` versus `%s/` (trailing `/`) on your PR

Reply:
Agree with @matzew, we should probably not add a trailing slash here. Golang's `url.URL.String()` doesn't: https://play.golang.org/p/cFaC63TxyXd

Reply:
Removed.